modules/guides/pages/serverless/lambda.adoc

```diff
@@ -141,46 +112,40 @@ output:
 
 === go1.x on x86_64
 
-Grab an archive labelled `benthos-lambda` from the https://github.com/{project-github}/releases[releases page^]
+Grab an archive labelled `redpanda-connect-lambda` from the https://github.com/{project-github}/releases[releases page^]
 page and then create your function:
 
-[source,sh]
+[source,sh, subs="attributes+"]]
 ----
 LAMBDA_ENV=`cat yourconfig.yaml | jq -csR {Variables:{BENTHOS_CONFIG:.}}`
 aws lambda create-function \
   --runtime go1.x \
-  --handler benthos-lambda \
+  --handler redpanda-connect-lambda \
   --role benthos-example-role \
-  --zip-file fileb://benthos-lambda.zip \
+  --zip-file fileb://redpanda-connect-lambda_{tagged-version}_linux_amd64.zip \
   --environment "$LAMBDA_ENV" \
   --function-name benthos-example
 ----
 
-There is also an example https://github.com/{project-github}/tree/main/resources/serverless/lambda/benthos-lambda-sam.yaml[SAM template^] and
-https://github.com/{project-github}/tree/main/resources/serverless/lambda/benthos-lambda.tf[Terraform resource^] in the repo to copy from.
-
-=== provided.al2 on amd64
+=== provided.al2 on arm64
 
-Grab an archive labelled `benthos-lambda-al2` for `arm64` from the https://github.com/{project-github}/releases[releases page^]
+Grab an archive labelled `redpanda-connect-lambda-al2` for `arm64` from the https://github.com/{project-github}/releases[releases page^]
 page and then create your function (AWS CLI v2 only):
 
-[source,sh]
+[source,sh, subs="attributes+"]
 ----
 LAMBDA_ENV=`cat yourconfig.yaml | jq -csR {Variables:{BENTHOS_CONFIG:.}}`
 aws lambda create-function \
   --runtime provided.al2 \
   --architectures arm64 \
   --handler not.used.for.provided.al2.runtime \
   --role benthos-example-role \
-  --zip-file fileb://benthos-lambda.zip \
+  --zip-file fileb://redpanda-connect-lambda-al2_{tagged-version}_linux_arm64.zip \
   --environment "$LAMBDA_ENV" \
   --function-name benthos-example
 ----
 
-There is also an example https://github.com/{project-github}/tree/main/resources/serverless/lambda/benthos-lambda-al2-sam.yaml[SAM template^] and
-https://github.com/{project-github}/tree/main/resources/serverless/lambda/benthos-lambda-al2.tf[Terraform resource^] in the repo to copy from.
-
-Note that you can also run `benthos-lambda-al2` on x86_64, just use the `amd64` zip instead.
+Note that you can also run `redpanda-connect-lambda-al2` on `x86_64`, just use the `amd64` zip instead.
```

Comment:
Also, is the information from Lines 166 onwards still correct?

Reply:
Yes I believe so